CDCgov/prime-fhir-converter
# Allow users to import library

This PR makes changes to the build scripts to allow external users to include the library in their Gradle scripts. We use the Gradle SourceControl method until we are able to officially publish the JAR to a binary repo. Alternatively, the JAR and fat JAR are generated by the build and can be provided in the releases section of the Github repo.

Changes:
* Changed the build scripts to correctly supply compile time libraries to the external application
* Added building the source and fat JAR
* Added documentation on how to include the library

Tests:
1. Run `./gradlew build` and verify the fat JAR is in the `build/libs` folder
2. Create a new Java application and include the library as stated in the instructions. Verify you can invoke the library functions.
